easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -64,7 +64,7 @@
 from easybuild.framework.easyconfig.parser import DEPRECATED_PARAMETERS, REPLACED_PARAMETERS
 from easybuild.framework.easyconfig.parser import EasyConfigParser, fetch_parameters_from_easyconfig
 from easybuild.framework.easyconfig.templates import template_constant_dict
-
+from easybuild.framework.easyconfig.templates import TEMPLATE_CONSTANTS
```

Comment:
join this with the `import` line above, leave two blank lines under last `import` (that's intentional)